File: gherkinexecutor/Feature_Examples/TemperatureCalculationInternal.java
package gherkinexecutor.Feature_Examples;
import java.util.*;
class TemperatureCalculationInternal{
     Integer f;
     Integer c;
     String notes;
     
    public static String toDataTypeString() {
        return "TemperatureCalculationInternal {"
        +"Integer " 
        +"Integer " 
        +"String " 
            + "} "; }  
    TemperatureCalculation toTemperatureCalculation() {
        return new TemperatureCalculation(
        String.valueOf(f)
        ,String.valueOf(c)
        ,notes
        ); }
    public TemperatureCalculationInternal(
        Integer f
        ,Integer c
        ,String notes
        )  {
        this.f = f;
        this.c = c;
        this.notes = notes;
        }
    @Override
    public boolean equals (Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        TemperatureCalculationInternal _TemperatureCalculationInternal = (TemperatureCalculationInternal) o;
         return 
                ( _TemperatureCalculationInternal.f.equals(this.f))
                 && ( _TemperatureCalculationInternal.c.equals(this.c))
                 && ( _TemperatureCalculationInternal.notes.equals(this.notes))
             ;  }
    @Override
    public String toString() {
        return "TemperatureCalculationInternal {"
        +"f = " + f + " "
        +"c = " + c + " "
        +"notes = " + notes + " "
            + "} " + "\n"; }  
    }

File: gherkinexecutor/Feature_Examples/TemperatureCalculationInternal.py
class TemperatureCalculationInternal:
    def __init__(self, f, c, notes):
        self.f = f
        self.c = c
        self.notes = notes

    @staticmethod
    def to_data_type_string():
        return "TemperatureCalculationInternal { Integer Integer String }"

    def to_temperature_calculation(self):
        return TemperatureCalculation(str(self.f), str(self.c), self.notes)

    def __eq__(self, other):
        if self is other:
            return True
        if other is None or not isinstance(other, TemperatureCalculationInternal):
            return False
        return self.f == other.f and self.c == other.c and self.notes == other.notes

    def __str__(self):
        return f"TemperatureCalculationInternal {{ f = {self.f} c = {self.c} notes = {self.notes} }}\n"

# Assuming TemperatureCalculation class is defined elsewhere

